STANDING_PERSON_HEIGHT: 100
OUTOFBED_LEFT: 240
OUTOFBED_RIGHT: 460
CUPX: 320
CUPY: 240
CUPR: 40
SENSITIVITY: 350
CUPTHRESHOLD: 230
CUPTHRSCOUNT: 30
